tap_pipedrive/stream.py

```diff
@@ -106,6 +105,7 @@ def state_is_newer_or_equal(self, current_state):
 
     def write_record(self, row):
         if self.record_is_newer_equal_null(row):
+            #logger.info('DEAL WRITTEN: {}, {}'.format(row['item_id'], row['log_time']))
```

Comment:
FWIW This might be a good place to use `logger.debug`? That's off by default but can be easily flipped on during development.

Reply:
ah, I meant to delete that line. I was just using it to print things out on my side for spot checks.